app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -99,29 +109,33 @@
         public int searchHintText = android.R.string.search_go;
     }
 
-    private static class WithPositionAdapter extends ArrayAdapter<Pair<String, Integer>> {
+    private static class WithPositionAdapter extends ArrayAdapter<Integer> {
         @LayoutRes
         final int _layout;
         final LayoutInflater _inflater;
         final DialogOptions _dopt;
-        final List<Pair<String, Integer>> _filteredItems;
+        final List<Integer> _filteredItems;
```

Comment:
Simplified the Adapter to just use List<Integer> instead of the pair, which was unnecessary